sktime/transformations/panel/tsfresh.py

```diff
@@ -616,9 +616,6 @@ def fit_transform(self, X, y=None):
                 then the return is a `Panel` object of type `pd-multiindex`
                 Example: i-th instance of the output is the i-th window running over `X`
         """
-        self.reset()
```

Comment:
In `fit()` there is:

```python
        # skip everything if fit_is_empty is True and we do not need to remember data
        if self.get_tag("fit_is_empty") and not self.get_tag("remember_data", False):
            self._is_fitted = True
            return self
```

Why is this not needed in `fit_transform()`?

Reply:
Regardless of whether `fit` is empty or not we have to return the transformed data, we can't just early exit.

Reply:
> Regardless of whether fit is empty or not we have to return the transformed data, we can't just early exit.

True, but that is just the return self. 

The check could remain

Reply:
```
if self.get_tag("fit_is_empty") and not self.get_tag("remember_data", False):
            self._is_fitted = True
```

Reply:
we `self._is_fitted = True` at the end anyway, no need to do it twice